reV/handlers/resource.py

```diff
@@ -305,7 +327,7 @@ def h5(self):
         return h5
 
     @property
-    def dsets(self):
```

Comment:
Thoughts on maintaining both  for backwards compatibility? I dont think keeping two aliases around would be a bad idea, especially when its a minor issue like abbreviation.

Reply:
good call. I'll add dsets back in